### ISSUE TITLE
Connection to 'wss://ws-pusher_cluster.pusher.com/....  failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVED

### ISSUE THREAD

**Issue by dejooyelese** [HackAfro/vue-event-scheduler#5]
Has anyone been able to get this program to work using a pusher account? I am getting refused connections and the error:

webSocket connection to 'wss://ws-pusher_cluster.pusher.com/app/PUSHER_KEY?protocol=7&client=js&version=4.4.0&flash=false' failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVED

I have tried both using the .env file and hardcoding making a new pusher as documented in the "Getting Started" of the pusher dashboard but not making any progress. Help Please.

**Comment by HackAfro** [comment]
Apologies for the errors you're facing. Are you sure you were connected to the internet when trying? If your errors persist, can you try cloning the repository and running the project.

Let me know how this goes.

**Comment by dejooyelese** [comment]
Yes I am connected to the internet, no cloning the repository and running it with my Pusher details in the .env does not solve the problem.

![Capture](https://user-images.githubusercontent.com/35816658/55193148-c9c51e00-516b-11e9-9ff1-8314a36ce988.PNG)

**Comment by HackAfro** [comment]
I can see in the errors in the screenshot that you haven't replaced the placeholder `PUSHER_KEY` value with your actual pusher key. Please ensure that you've updated the values in the `.env` file and the `App.vue` file.

**Comment by dejooyelese** [comment]
Okay - thank you! I didn't realize App.vue also had PUSHER_KEY values that needed to be replaced.

New Problem when creating a test event: I do not understand the error so therefore don't know where to start trouble shooting. Help is again appreciated
![Capture](https://user-images.githubusercontent.com/35816658/55253425-ec624000-521a-11e9-8075-6f885f974914.PNG)

**Comment by dejooyelese** [comment]
Server Terminal Error:

![Capture](https://user-images.githubusercontent.com/35816658/55254869-a6a77680-521e-11e9-9a12-ec2a5b609ff1.PNG)

**Comment by HackAfro** [comment]
Might be far fetched but please look at your `.env` file and ensure it is similar to the snippet below:

```
PUSHER_ID=app-id
PUSHER_KEY=app-key
PUSHER_SECRET=app-secret
PUSHER_CLUSTER=cluster
```

**Comment by dejooyelese** [comment]
![Capture](https://user-images.githubusercontent.com/35816658/55265661-e67e5600-523e-11e9-99bb-2da27a275308.PNG)

**Comment by dejooyelese** [comment]
I fixed the .env file to match the wording on the server.js file (I think the documentation in the README is different from what the file should be). Now there is no error and the submit button works but nothing appears on the screen. There is also no console errors.

It seems I am directly in line with the errors with this other user so I will be monitoring that post as well as this one. Thanks for all the help!
https://github.com/HackAfro/vue-event-scheduler/issues/4

**Comment by dejooyelese** [comment]
New File Setup:
![Capture](https://user-images.githubusercontent.com/35816658/55265931-07937680-5240-11e9-99ba-cf045f812142.PNG)
